cmd/soroban-cli/src/commands/contract/alias/remove.rs

```diff
@@ -0,0 +1,72 @@
+use std::fmt::Debug;
+
+use clap::{command, Parser};
+
+use crate::commands::{config::network, global};
+use crate::config::locator;
+use crate::print::Print;
+
+#[derive(Parser, Debug, Clone)]
+// #[command(group(
+//     clap::ArgGroup::new("wasm_src")
+//         .required(true)
+//         .args(&["wasm", "wasm_hash"]),
+// ))]
+#[group(skip)]
+pub struct Cmd {
+    #[command(flatten)]
+    pub config_locator: locator::Args,
+
+    #[command(flatten)]
+    network: network::Args,
+
+    /// The contract alias that will be removed.
+    pub alias: String,
+}
+
+#[derive(thiserror::Error, Debug)]
+pub enum Error {
+    #[error(transparent)]
+    Locator(#[from] locator::Error),
+
+    #[error(transparent)]
+    Network(#[from] network::Error),
+
+    #[error("no contract found with alias `{alias}`")]
+    NoContract { alias: String },
+}
+
+impl Cmd {
+    #[allow(clippy::unused_async)]
+    pub async fn run(&self, global_args: &global::Args) -> Result<(), Error> {
+        let print = Print::new(global_args.quiet);
+        let alias = &self.alias;
+        let network = self.network.get(&self.config_locator)?;
+
+        print.infoln(format!(
+            "Network passphrase: {passphrase}",
+            passphrase = network.network_passphrase
```

Comment:
It feels random that this gets printed out. As a user I'm not sure what it means. I think we need to tell the narrative a little to take the user on the journey about how these things relate:

```
ℹ️ Contract alias '<alias>' references CCYBGMETD5TYZJ2ZBT3JBBCWODYJJWP274JA4LX4S7JGNUNQPXZBMNUY on network 'Test SDF Network ; September 2015'
✅ Contract alias '<alias>' has been removed
```